sample-js-browser-app/build.gradle

```diff
@@ -1,9 +1,12 @@
+import com.badoo.reaktive.configuration.Target
 import com.badoo.reaktive.dependencies.Deps
 
-plugins {
-    id 'org.jetbrains.kotlin.js'
+if (!Target.shouldDefineTarget(project, Target.JS)) {
+    return
 }
 
+plugins.apply("org.jetbrains.kotlin.js")
```

Comment:
I think having a list of modules to compile on macOS is better than using `apply` instead of `plugins` as autocompletion might not work properly in the script.

Reply:
My intention was to unify build/yaml scripts for consistency (e.g. we have similar logic in `sample-ios-app` and `sample-macos-app` modules), and also we might add Windows targets in future. But if you believe the current approach is better, I will revert the changes. Thanks!

Reply:
Revert please, we do not have too much modules to maintain and we are more explicit this way.